### PR TITLE
Add AddFreeTime command

### PR DESCRIPTION
**Pull request by larainezo**
image

### CODE REVIEW

**Review by PallonCX — APPROVED** [review]
LGTM, it will be good if some visuals (i.e. screenshots) are shown to demonstrate the changes introduced to the application.

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public class AddCommand extends Command {
             + PREFIX_ROOMNUMBER + "sw-01-01 "
             + PREFIX_TELEGRAM + "johnDoe "
             + PREFIX_BIRTHDAY + "12/12/2000 "
-            + PREFIX_FREETIMETAG + "Mon:1pm-2pm";
+            + PREFIX_FREETIMETAG + "Mon:1300-1400";
```
Comment (donwong2308) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:41]: Thanks for picking up my error:)

File: src/main/java/seedu/address/logic/commands/AddTimeCommand.java [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+import seedu.address.model.tag.FreeTimeTag;
+
+/**
+ * Adds a free time to an existing person in the address book.
+ */
+public class AddTimeCommand extends Command {
+
+    public static final String COMMAND_WORD = "addTime";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a free time to the person identified "
```
Comment (donwong2308) [inline src/main/java/seedu/address/logic/commands/AddTimeCommand.java:33]: Perhaps you can add an example of how to use the add time command in the PR description

**Comment by larainezo** [comment]
> LGTM, it will be good if some visuals (i.e. screenshots) are shown to demonstrate the changes introduced to the application.

Just added a screenshot! Thanks!

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 3]
```diff
@@ -54,7 +54,7 @@ public class EditCommand extends Command {
             + PREFIX_PHONE + "91234567 "
             + PREFIX_EMAIL + "johndoe@example.com";
 
-    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
+    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited person: %1$s";
```
Comment (donwong2308) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:57]: Good catch on the typo!

**Review by donwong2308 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/AddTimeCommand.java [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Person> lastShownList = model.getFilteredPersonList();
+
+        if (index.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        }
+
+        Person personToEdit = lastShownList.get(index.getZeroBased());
+        Person editedPerson = createEditedPerson(personToEdit, editPersonDescriptor);
```
Comment (nyankoclaws) [inline src/main/java/seedu/address/logic/commands/AddTimeCommand.java:65]: Maybe can add a few basic checks..?

**Review by nyankoclaws — APPROVED** [review]
LGTM

**Review by JustWeiHao — APPROVED** [review]
LGTM

**Pull request merged**